app/src/main/java/com/example/diceroller/MainActivity.kt

```diff
@@ -37,18 +42,15 @@ fun DiceRollerApp(){
 }
 
 @Composable
-fun DiceWithButtonAndImage(
-    modifier: Modifier = Modifier
+fun DiceWithButtonAndImage(modifier: Modifier = Modifier
     .fillMaxSize()
     .wrapContentSize(Alignment.Center))
 {
-    Column(
-        modifier = modifier,
-        horizontalAlignment = Alignment.CenterHorizontally
-    ) {
-        Button(onClick = { /*TODO*/ })
-        {
-            Text(stringResource(R.string.roll))
-        }
-    }
-}
\ No newline at end of file
+    Column(modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally)
+    {
+        Image(
+            painter = painterResource(R.drawable.dice_1)
+            ,contentDescription = "1")
```

Comment:
The comma is in the wrong line, and same thing with the parentheses: 
```suggestion
        Image(
            painter = painterResource(R.drawable.dice_1),
            contentDescription = "1"
            )
```